uzdevums_6.py

```diff
@@ -0,0 +1,11 @@
+print("Ievadiet skaitli ar kuru velaties parbaudit vai sakitlis dalas ar 5 vai 7 :")
+n=input()
```

Comment:
Labāk vienā vietā pārveidot no simbolu virknes uz skaitli:
```suggestion
n=int(input())
```